app/views/devise/sessions/new_with_providers.html.erb

```diff
@@ -0,0 +1,35 @@
+<% if @local_account %>
+  <%= render "devise/shared/form" %>
+<% end %>
+<div class="grid gap-2">
+  <% if @local_account %>
+    <%= render "devise/shared/links" %>
+    <%= button_to t(".return_button"),
+    new_user_session_path,
+    method: :get,
+    params: {
```

Comment:
We don't need to use this params: {}. We can just add the `local: true` to the `new_user_session_path` above

Reply:
Disregard and just leave the params in as this would require a route change

Reply:
As discussed, please see 2817030b69fb2ca9cedc353957f627a8c9f57a51.